distribution/pom.xml

```diff
@@ -31,6 +31,10 @@
     <artifactId>keycloak-distribution-parent</artifactId>
     <packaging>pom</packaging>
 
+    <properties>
+        <keycloak.provisioning.xml>server-provisioning.xml</keycloak.provisioning.xml>
+    </properties>
+
```

Comment:
Didn't you included this by accident?

Reply:
Sorry, as long as it's ok to cover both independent changes in single PR, it looks ok.